thesis-fe/package.json

```diff
@@ -9,6 +9,7 @@
     "apollo-link-http": "^1.5.7",
     "emotion": "^9.2.12",
     "graphql": "^14.0.2",
+    "jquery": "^3.3.1",
```

Comment:
jQuery?

Reply:
it was needed for a component i was using from a library, but actually i removed that component so i should remove it from the json file. Doing it now